examples/gocean/eg1/opencl_transformation.py

```diff
@@ -33,50 +33,42 @@
 # -----------------------------------------------------------------------------
 # Authors: S. Siso, STFC Daresbury Lab
 
-'''A simple test script showing the introduction of OpenCL with PSyclone.
-In order to use it you must first install PSyclone like so:
+''' Module providing a transformation script that converts the Schedule of
+    each Invoke to use OpenCL. '''
 
- >>> pip install --user psyclone
 
-(or see the Getting Going section in ../../psyclone.pdf.) Once PSyclone is
-installed, this script may be run by doing:
+def trans(psy):
+    '''
+    Transformation routine for use with PSyclone. Applies the OpenCL
+    transform.
```

Comment:
"Converts any global-variable accesses into kernel arguments and then applies the OpenCL transformation to the PSy layer"

Reply:
Done